storage/storage.cpp

```diff
@@ -985,4 +986,16 @@ bool Storage::IsNodeDownloaded(TCountryId const & countryId) const
   }
   return false;
 }
+
+bool Storage::IsPointCovered(m2::PointD const & position,
+                             CountryInfoGetter const & countryInfoGetter) const
+{
+  TCountryId countryId = countryInfoGetter.GetRegionFile(position);
+  if (countryId == string())
+    return false;
+
+  vector<TCountryId> localMaps;
```

Comment:
Do you really need to put *ALL* maps into a vector to find a specific map? Especially in presence of a m_localFiles map?